pillar/backups/mitx.sls

```diff
@@ -10,6 +10,13 @@
 {% set efs_id = 'fs-1f27ae56' %}
 {% endif %}
 
+schedule:
```

Comment:
This one isn't necessary since the credentials would get updated as part of the backup orchestrate state and we want it to run nightly.